odxtools/diaglayer.py

```diff
@@ -591,6 +594,27 @@ def not_inherited_fn(parent_ref: ParentRef) -> List[str]:
 
         return self._compute_available_objects(get_local_objects_fn, not_inherited_fn)
 
+    def _add_import_ref(self, ecu_shared_datas) ->  None:
```

Comment:
I guess that after #285, the IMPORT-REF mechanism is implemented. What can/should be added to `Database` and `DiagLayerContainer` though, are wrapper attributes for the different kinds of `DiagLayers` (analogos to `.ecus` or `.protocols`)